mmv1/products/cloudbuild/terraform.yaml

```diff
@@ -49,6 +49,11 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       - !ruby/object:Provider::Terraform::Examples
         name: "cloudbuild_trigger_manual"
         primary_resource_id: "manual-trigger"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "cloudbuild_trigger_repo"
+        min_version: beta
+        primary_resource_id: "repo-trigger"
+        skip_test: true
```

Comment:
Can we amend this configuration to generate a working test? If that's impossible, what's the blocker?